src/heroku/heroku-dispatch.js

```diff
@@ -0,0 +1,16 @@
+import { dispatchProcesses } from "../workers/job-processes";
```

Comment:
I'd rather this just be in src/workers/dispatch-processes.js or something.  I don't think this is specific to Heroku, and in the future it might be better not to suggest this would run by default in a Heroku deployment.

Reply:
cc @jeffm2001 FYI

i can def make the change this before we talk about merging, but will wait for now.

Reply:
Makes sense. That's where I originally had it, but moved it because I thought it be confusing for folks on AWS. But agree it's better to keep in one place.